doc/workflow.md

```diff
@@ -0,0 +1,66 @@
+# Workflow
+
+#### **kube-spawn-dir**
+`/var/lib/kube-spawn` by default, can be set with `--dir`
+
+#### **cluster-dir**
+`<kube-spawn-dir>/default` by default, can be set with `--cluster-name`
+
+## `create`
+
+* _check provided config_ (not implemented)
```

Comment:
Is there an issue about this? If not, we can remove this line, and add it only when it is actually implemented.

No dead code and no dead documentation ;)